src/Tests/BusinessLayer/SerialReportingTest.cpp

```diff
@@ -349,7 +349,8 @@ TEST_F(SerialReportingTest, sendDriverControlsTest) // TODO create function whic
                             driverControlsData_->pushToTalk(),
                             driverControlsData_->horn(),
                             driverControlsData_->reset(),
-                            driverControlsData_->aux()
+                            driverControlsData_->aux(),
+                            driverControlsData_->lap()
                           };
     Util::writeBoolsIntoArray(data, 8, driverInputs, 7);
```

Comment:
Shouldn't this also be increased to 8, like in the other file?
` writeBoolsIntoArray(packetPayload, 8, controlsArray, 8);`

Reply:
> Shouldn't this also be increased to 8, like in the other file?
> ` writeBoolsIntoArray(packetPayload, 8, controlsArray, 8);`

@mglyngberg and I tried setting this to 8 and it stopped working during testing. Tbh not sure why this works, but it does

Reply:
Even though the test does not verify the contents of the message it should have the same logic as the actual function.